shell/common/extensions/electron_extensions_client.cc

```diff
@@ -85,7 +82,9 @@ void ElectronExtensionsClient::InitializeWebStoreUrls(
 const extensions::PermissionMessageProvider&
 ElectronExtensionsClient::GetPermissionMessageProvider() const {
   NOTIMPLEMENTED();
-  return g_permission_message_provider.Get();
+
+  base::NoDestructor<ElectronPermissionMessageProvider> instance;
```

Comment:
wouldn't this leak if not `static` ?

Reply:
Yikes! Thank you for catching this. Fixed in d8d5c8209b2419dbe88cf91cc80fcff23f942de3